src/Compilers/CSharp/Test/Symbol/Compilation/GetSemanticInfoTests.cs

```diff
@@ -5027,6 +5027,51 @@ public void GetPreprocessingSymbolInfoForSymbolDefinedLaterInSource()
             Assert.False(symbolInfo.IsDefined, "must not be defined");
         }
 
+        [Fact, WorkItem(72907, "https://github.com/dotnet/roslyn/issues/72907")]
+        public void GetPreprocessingSymbolInfoOnUnrelatedDirective()
```

Comment:
>GetPreprocessingSymbolInfoOnUnrelatedDirective [](http://example.com/codeflow?start=20&length=46)

It looks like no tests were added for positive scenarios

Reply:
Other tests already cover the positive scenarios, that's why this test didn't focus on them. I will add the assertion for `#define` which we expect to return a non-null symbol